client/state/importer-nux/actions.js

```diff
@@ -8,9 +15,33 @@ import {
 	IMPORTER_NUX_URL_INPUT_SET,
 	IMPORT_IS_SITE_IMPORTABLE_START_FETCH,
 } from 'state/action-types';
+import { infoNotice, removeNotice } from 'state/notices/actions';
+import { loadmShotsPreview } from 'my-sites/importer/site-importer/site-preview-actions';
```

Comment:
Definitely. This was bothering me somewhat too. What do you think to moving it to `lib` since there's no actual state actions being dispatched here, just promises which could be used from an action just the same as any other promise.